RadixWallet/Features/P2PLinksFeature/Coordinator/P2PLinksFeature+Reducer.swift

```diff
@@ -91,15 +93,21 @@ public struct P2PLinksFeature: Sendable, FeatureReducer {
 		case .addNewConnectionButtonTapped:
 			state.destination = .newConnection(.init())
 			return .none
+
+		case let .removeButtonTapped(link):
+			state.destination = .removeConnection(.confirmRemoval(link: link))
+			return .none
+
+		case let .editButtonTapped(link):
+			state.destination = .updateName(.init(link: link))
+			return .none
 		}
 	}
 
 	public func reduce(into state: inout State, internalAction: InternalAction) -> Effect<Action> {
 		switch internalAction {
 		case let .loadLinksResult(.success(linksFromProfile)):
-			state.links = .init(
-				uniqueElements: linksFromProfile.map { P2PLinkRow.State(link: $0) }
-			)
+			state.links = .init(uniqueElements: linksFromProfile)
```

Comment:
Should be able o use `linksFromProfile.asIdentifiable()`, preferred to `.init(uniqueElements:` which can potentially crash.